NDB.Covid19/NDB.Covid19/Models/DailySummaryConfigurationDTO.cs

```diff
@@ -0,0 +1,12 @@
+﻿using System;
+namespace NDB.Covid19.Models
+{
+    public class DailySummaryConfigurationDTO
+    {
+        public Xamarin.ExposureNotifications.DailySummaryConfiguration DailySummaryConfiguration { get; set; }
```

Comment:
Please use static import for so long paths ;)

Reply:
Fixed